File: LatPrak2_1 (JAVA)/Memory.java
class Memory extends Hardware{
	//atribut
	private String frequency;
	private String memory_size;
	private String supports_cuda;

	//constructor
	Memory(){
	}

	//get & set untuk atribut frequency
	public void setFrequency(String frequency){
		this.frequency = frequency;
	}
		
	public String getFrequency(){
		return this.frequency;
	}

	//get & set untuk atribut memory size
	public void setMemory(String memory_size){
		this.memory_size = memory_size;
	}
		
	public String getMemory(){
		return this.memory_size;
	}

	//get & set untuk atribut support Cuda
	public void setSupportCuda(String supports_cuda){
		this.supports_cuda = supports_cuda;
	}
		
	public String getSupportCuda(){
		return this.supports_cuda;
	}
}

File: LatPrak2_2 (PY)/Memory.py
from Hardware import Hardware

class Memory(Hardware):
#kelas yang digunakan untuk mengimplementasikan kelas Memory

	#atribut
	__frequency = "";
	__memory_size = "";
	__support_cuda = "";

	#constructor
	def __init__(self, frequency = "", memory_size = "", support_cuda = ""):
		self.frequency = frequency
		self.memory_size = memory_size
		self.support_cuda = support_cuda		

	#get dan set untuk atribut frequency
	def setFrequency(self, frequency):
		self.frequency = frequency

	def getFrequency(self):
		return self.frequency
	
	#get dan set untuk atribut Memory size
	def setMemory(self, memory_size):
		self.memory_size = memory_size

	def getMemory(self):
		return self.memory_size

	#get dan set untuk atribut support cuda
	def setSupportCuda(self, support_cuda):
		self.support_cuda = support_cuda

	def getSupportCuda(self):
		return self.support_cuda
	
	
	